src/microbe_stage/editor/TolerancesEditorSubComponent.cs

```diff
@@ -188,6 +203,7 @@ public override void OnEditorSpeciesSetup(Species species)
         }
 
         UpdateToolTipStats();
+        UpdateAdditionalSliderInfo();
```

Comment:
Like here "AdditionalSliderInfo" is much more descriptive than "ToleranceInfo" in regards to what it actually is.